### DISCUSSION TITLE
💙💛 Free cloud, hosting, and add-ons for Ukrainian companies

### DISCUSSION THREAD

**Discussion by paula-w** [YetiForceCompany/YetiForceCRM#15786] [Announcement]
### To show our support for our friends in Ukraine we decided to remove all cloud, hosting, and add-on fees for all companies registered in Ukraine. 

If your company is registered in Ukraine and you’re currently using any of these products, or if you would like to start using them free of charge, please send us an email to [hello@yetiforce.com](mailto:hello@yetiforce.com).

The offer applies to the following products:
- [YetiForce Cloud [VPS]](https://yetiforce.com/en/marketplace/cloud)
- [YetiForce Hosting](https://yetiforce.com/en/marketplace/hosting-en)
- [YetiForce Addons](https://yetiforce.com/en/marketplace/addons)

![UA_EN](https://user-images.githubusercontent.com/13310141/157672862-44306a2c-3348-45d0-8372-a6f839c93c6e.png)

**Comment by waran70** [comment]
Playing in this platform politic games is dirty. Your announcement ridiculing own citizens. Why Ukrainians are above Poles? Or any other nation? By the way: https://en.wikipedia.org/wiki/Massacres_of_Poles_in_Volhynia_and_Eastern_Galicia

**Reply by paula-w** [reply]
We don’t condone any wars. 

We don’t live in the past and we don’t carry grudges. What matters is what’s happening now, namely, the russian invasion of Ukraine. We consider it genocide. 

Additionally, we are planning to disallow the use of YetiForce CRM in the russian federation. 

We’re not going to further engage in this type of discussion.
